nx.json

```diff
@@ -7,7 +7,7 @@
     "default": {
       "runner": "nx-cloud",
       "options": {
-        "cacheableOperations": ["build", "test", "lint", "e2e"],
+        "cacheableOperations": ["build", "test", "lint", "e2e", "docker-compose"],
```

Comment:
Why would `docker-compose` be cacheable?

Reply:
Without cachable docker-compose you cant run this target with dte

Reply:
Docker-compose is bootstrapping something, this is similar as I would want to cache the `serve` action and should not be cacheable afaik. I don't imagine what is cached in this case. Can you explain? Why should this command be executed using DTE, isn't it local only?

Reply:
You have setup pipeline with dte. You cant run target in dte that is not in cache target list. This target will not cache anything in outputs, but is required by nx. This is an nx architectural requirement.

In new commit I concatenate both targets e2e+docker-compose to one target with executor nx/run-commands

Reply:
I understand that, but why would you run `docker-compose` in CI (github action) in any case? Is this local only or not? DTE is only relevant for CI.